policyengine_canada/tests/gov/cra/tax/income/credits/gst_credit/person_credits.yaml

```
@@ -0,0 +1,193 @@
+- name: Base amount for family of 3
+  period: 2023
+  input:
+    people:
+      parent:
+        gst_credit_category: HEAD
+      c1:
+        gst_credit_category: SPOUSE
+      c2:
+        gst_credit_category: CHILD
+    household:
+      members: [parent, c1, c2]
+  output:
+    gst_credit_base: 306 + 306 + 161
+
+- name: Number of dependent children
+  period: 2023
+  input:
+    people:
+      parent:
+        gst_credit_category: HEAD
+        is_dependent: false
+        age: 55
+      c1:
+        gst_credit_category: CHILD
+        is_dependent: true
+        age: 16
+      c2:
+        gst_credit_category: CHILD
+        is_dependent: true
+        age: 17
+    household:
+      members: [parent, c1, c2]
+  output:
+    gst_credit_dependent_children: 2
+    gst_credit_base: 306 + 161 + 161
```

If I'm understanding the single-parent bit correctly, per SPSD/M's clarification
```suggestion
    gst_credit_base: 306 + 306 + 161
```